medical_pathology/tests/test_medical_pathology.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from psycopg2 import IntegrityError
+
+from openerp.tests.common import TransactionCase
+from openerp.exceptions import ValidationError
+
+
+class TestMedicalPathology(TransactionCase):
+
+    def setUp(self):
+        super(TestMedicalPathology, self).setUp()
+        self.pathology_1 = self.env.ref(
```

Comment:
Hmmm yeah but it's 2 of 2, which is 100% of the test cases. Seems worth it IMO - saves 3 lines